testCustomizations.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
Can you also delete the old json files that were combined into this one?

Reply:
Oops. Yep, that was meant to be included here. I'll get that in the next PR 👍